### ISSUE TITLE
Tabbed pane doesn't wrap (and other things)

### ISSUE THREAD

**Issue by sorahn** [Wyrrrd/IndustrialDisplayPlates#1]
Hey first off, thanks for picking this up :)  We ran into a couple of issues we had pinged Deadlock about but he wasn't really interested in fixing them, and decided to offer up the mod for anyone who wanted it.

The biggest one is when playing with a mod that adds lots of sub groups, the tabbed pane display doesn't wrap so the tabs start to get really skinny.   (easy to see with Pyanodons)

The second one is that when you filter with the search it doesn't automatically remove tabs that have no results.

Lastly, maybe more of a feature request (happy to move this into a separate issue), but more plates made out of more things (u235, titanium if you're playing FE+, etc etc)

Thanks again :D

**Comment by ghost** [comment]
The first one is not so easy to fix. Vanilla went with what I think is a really annoying solution - have a special "icon tabs" pane by default if there are six groups or less, but create an entirely different gui using buttons if there are more than six. We don't even have access to the "icon tab" elements used for the former, I had to fake it, and it pissed me off to have to implement two different guis on the same thing just because that is what vanilla did. To be honest I think my mistake was trying to emulate the vanilla GUI in the first place. If I were interested in fixing the mod to cope with anything between 4 and a billion modded tabgroups then I think some other design would be more appropriate, such as one long scrollable vertical list, with the "tabs" down the side as buttons which jump the scrollbar to that subgroup.

The second issue is related to the comment about about not being able to create real icon tab GUI elements. Vanilla disables a tab when there are no search hits. We can't do that because it's a fake icon tab. I did start experimenting with creating fake greyed out icons for every item group but lost patience with it.

Wyrrd has talked about using chooser gui elements and the native item selection menu instead which would solve those problems but I couldn't find a way of doing it that didn't feel clunky with windows popping out of windows etc.

TL;DR - the real solution to these issues is to not bother with vanilla style because the modding API only gives us a subset of the GUI elements used by vanilla.

**Comment by Wyrrrd** [comment]
Moved additional material request to #13.

**Comment by Wyrrrd** [comment]
Wontfix, see [README.md](https://github.com/Wyrrrd/IndustrialDisplayPlates-Archive#readme) (sorry).
